### PR TITLE
Added script to enable pr checks for yuptoo

### PR DESCRIPTION
**Pull request by Aurobinda55**
This pr is to add the pr check script in yup

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -3,4 +3,26 @@
 set -exv
 
 pipenv install -d
-flake8
\ No newline at end of file
+flake8
```
Comment (r14chandra) [inline pr_check.sh:6]:
```
set -exv
pipenv install -d
flake8
```
Do we need these three lines of code?

File: pr_check.sh [hunk 2]
```diff
@@ -3,4 +3,26 @@
...
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+# Build the image and push to quay
+source $CICD_ROOT/build.sh
+
+# Deploy to an ephemeral namespace for testing
+source $CICD_ROOT/deploy_ephemeral_env.sh
+
+# Run smoke tests with ClowdJobInvocation
+source $CICD_ROOT/cji_smoke_test.sh
```
Comment (san7ket) [inline pr_check.sh:28]:
```suggestion
source $CICD_ROOT/cji_smoke_test.sh
source $CICD_ROOT/post_test_results.sh
```

**Comment by san7ket** [comment]
@patilsuraj767 @kgaikwad Can we give InsightsDroid Bot write access to Repo so that it can post test results status in every PR

**Comment by san7ket** [comment]
Also lets handle flake8 before the deploy is triggered.

**Comment by san7ket** [comment]
We are Missing build.yaml from app-sre repo. We need it to spin up pr check Job

File: pr_check.sh (hunk 1 above)
Comment (Aurobinda55) [reply pr_check.sh]: removed these codes

**Comment by san7ket** [comment]
@patilsuraj767 @kgaikwad can we merge this ?

**Comment by r14chandra** [comment]
Thanks @Aurobinda55 and @san7ket! Merging this PR.

**Pull request merged**